tests/set-auth-role.ts

```diff
@@ -0,0 +1,16 @@
+import { AuthorizationRoles } from '@/enums'
+import * as Actions from './unit/test-data/AuthorizedActionsLists'
+
+/** Sets the authorized actions (aka permissions) according to Auth Role. */
+export function setAuthRole (store: any, authRole: AuthorizationRoles = null): void {
```

Comment:
I initially declared the store in this function but it caused conflicts with store.resourceModel, which makes no sense whatsoever. Anyway, this works nicely.